locator/tablets.cc

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (C) 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include "locator/tablet_replication_strategy.hh"
+#include "locator/tablets.hh"
+#include "types/types.hh"
+#include "types/tuple.hh"
+#include "types/set.hh"
+#include "utils/hash.hh"
+#include "db/system_keyspace.hh"
+#include "cql3/query_processor.hh"
+#include "cql3/untyped_result_set.hh"
+#include "replica/database.hh"
+
+#include <seastar/core/coroutine.hh>
+#include <seastar/coroutine/maybe_yield.hh>
+
+namespace locator {
+
+seastar::logger tablet_logger("tablets");
+
+std::ostream& operator<<(std::ostream& out, const tablet_replica& r) {
+    return out << r.host << ":" << r.shard;
```

Comment:
@tchaikov would ask for a fmt::formatter specialization as this seems to be the new convention.

@tchaikov I don't think it's documented anywhere yet.
How about adding something under docs/dev?
There should probably be also a higher-level doc pointing to it, though I'm not sure which.

Reply:
Is this a blocker?

Reply:
@bhalevy @tgrabiec yeah, but we can always fix it latter. i don't think it's a blocker. regarding the documentation, will come up with a proposal in this week. it will likely based on the issue of #13245 .